app/main.py

```diff
@@ -2,0 +1,7 @@
+def copy_file(command: str) -> None:
+    command_ls = command.split()
+    if len(command_ls) == 3 and command_ls[0] == "cp" and command_ls[1] != command_ls[2]:
+        with open(command_ls[1], "r") as r_file, open(command_ls[2], "w") as w_file:
```

Comment:
Code Style: Variable names 'r_file' and 'w_file' are not descriptive enough. Consider using more meaningful names, like 'source_file' and 'destination_file'.